tests/coverage/unreachable/variant/main.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
```

Comment:
Please add a description, e.g.: This test checks that code coverage on a match statement with an enum behaves correctly